app/javascript/modules/listings/GenericDirectory.tsx

```diff
@@ -81,20 +82,22 @@ export const GenericDirectory = (props: RentalDirectoryProps) => {
               {props.findMoreActionBlock()}
               {filters &&
                 additionalView(
-                  listings.additional,
+                  isSalesFcfsEnabled
+                    ? listings.additional
+                    : removeSalesFcfsListings(listings.additional),
                   props.directoryType,
                   props.getSummaryTable,
                   hasFiltersSet,
                   isSalesFcfsEnabled
                 )}
               {upcomingLotteriesView(
-                listings.upcoming,
+                isSalesFcfsEnabled ? listings.upcoming : removeSalesFcfsListings(listings.upcoming),
                 props.directoryType,
                 props.getSummaryTable,
                 isSalesFcfsEnabled
               )}
               {lotteryResultsView(
-                listings.results,
+                isSalesFcfsEnabled ? listings.results : removeSalesFcfsListings(listings.results),
```

Comment:
question: Do we need to filter out `additional`, `upcoming`, and `results` list based on the feature flag? We should only show sales fcfs listings in the `openListingsView` even when they toggle is enabled.

Reply:
Oh I guess probably not! How are they being limited to the open listings view?

Reply:
This may provide more context: https://arc.net/l/quote/vaehxmwr

Reply:
so we just need to toggle the fcfsSales view I think!

Reply:
I don't think fcfs view is supposed to be different that open listings view

Reply:
@cade-exygy @alulabeshue-sfgov I'll put this in draft until cade's pr is merged!